src/ontology/mondo.Makefile

```diff
@@ -653,10 +653,8 @@ update_deprecated_mappings: $(ALL_SOURCES_DEPRECATED_PATTERNS)
 	$(MAKE) NORM && mv NORM $(SRC)
 
 deprecated_annotation_merging:
-	sed -i 's/source="MONDO:equivalentObsolete",\ source="MONDO:obsoleteEquivalentObsolete"/source="MONDO:obsoleteEquivalentObsolete"/g' mondo-edit.obo || true
-	sed -i 's/source="MONDO:equivalentObsolete",\ source="MONDO:equivalentTo"/source="MONDO:equivalentObsolete"/g' mondo-edit.obo || true
-	#(.*)source="MONDO:equivalentObsolete"(.*)source="MONDO:obsoleteEquivalentObsolete"(.*) // 
-	echo "NOTE: There are still some broken cases that need to be manually fixed. Search for `quivalent.*quivalent` (no E) in case sensitive regex mode in Atom or Visual Studio"
+	sed -i '' 's/\(.*\)source="MONDO:equivalentObsolete"\(.*\)source="MONDO:obsoleteEquivalentObsolete"\(.*\)/\1source="MONDO:obsoleteEquivalentObsolete"\2\3/g' mondo-edit.obo || true
+	sed -i '' 's/, }/}/g' mondo-edit.obo || true
```

Comment:
I've since replaced `sed -i ''` with `sed -i` for consistency. Note that on my Macbook, I needed `sed -i ''` to get it to work ([stackoverflow](https://stackoverflow.com/a/5694430/5258518)).